parse/set.c

```diff
@@ -554,15 +561,15 @@ enum CommandResult parse_set(struct Buffer *buf, struct Buffer *s,
       s->dptr++;
     }
 
-    if (!bool_or_quad && (inv || (unset && prefix)))
+    if (!invertible && (inv || (unset && prefix)))
     {
       if (data == MUTT_SET_SET)
       {
-        buf_printf(err, _("Prefixes 'no' and 'inv' may only be used with bool/quad variables"));
+        buf_printf(err, _("Prefixes 'no' and 'inv' may only be used with bool/quad/number variables"));
```

Comment:
Hm, I'm probably doing something stupid but how can I use `set no` with a number config?

Reply:
Oh right, I haven't looked into `no`. `inv` is how `toggle` is implemented:
https://github.com/neomutt/neomutt/blob/f10c0f0dd82d97f88fe54efc1e9aad2c1066d407/commands.c#L1700

Reply:
A `no` prefix is the same as `unset`:
https://github.com/neomutt/neomutt/blob/f10c0f0dd82d97f88fe54efc1e9aad2c1066d407/commands.c#L1719

We can probably make `unset wrap` be equivalent to `set wrap 0`.

Reply:
Now you can unset a numeric value (or setting it with a `no` prefix) to turn it into a 0 :)